mirdata/datasets/tonas.py

```diff
@@ -0,0 +1,409 @@
+        3. 20 Martinete 2
+
+    This collection was built in the context of a study on similarity and style classification of flamenco a cappella
+    singing styles (Tonas) by the flamenco expert Dr. Joaquin Mora, Universidad de Sevilla.
+
+    We refer to (Mora et al. 2010) for a comprehensive description of the considered styles and their musical
+    characteristics. All 72 excerpts are monophonic, their average duration is 30 seconds and there is enough
+    variability for a proper evaluation of our methods, including a variety of singers, recording conditions,
+    presence of percussion, clapping, background voices and noise. We also provide manual melodic transcriptions,
+    generated by the COFLA team and Cristina López Gómez.
+
+    The annotations are represented by specifying the value (in this case, Notes and F0) at the related timestamps.
+    TONAS' note and F0 annotations also have "Energy" information, which refers to the average energy value through
+    all the frames in which a note or a F0 value is comprised.
+
+    Using this dataset:
+    TONAS dataset can be obtained upon request. Please refer to this link: https://zenodo.org/record/1290722 to
+    request access and follow the indications of the .download() method for a proper storing and organization
+    of the TONAS dataset.
+    When TONAS is used for academic research, we would highly appreciate if scientific publications of works
+    partly based on the TONAS dataset quote the following publication:
+
+"""
+import csv
+import os
+from typing import cast, TextIO, Tuple, Optional
+
+import librosa
+import numpy as np
+
+from mirdata import jams_utils
+from mirdata import core
+from mirdata import annotations
+from mirdata import io
+
+
+BIBTEX = """
+@dataset{cofla_computational_analysis_of_flamenco_2013_1290722,
+    author       = {COFLA (COmputational analysis of FLAmenco music) team},
+    title        = {{TONAS: a dataset of flamenco a cappella sung 
+                     melodies with corresponding manual transcriptions}},
+    month        = mar,
+    year         = 2013,
+    publisher    = {Zenodo},
+    version      = {1.0},
+    doi          = {10.5281/zenodo.1290722},
+    url          = {https://doi.org/10.5281/zenodo.1290722}
+}
+@inproceedings{inproceedings,
+    author = {Mora, Joaquin and Gómez, Francisco and Gómez, Emilia
+              and Borrego, Francisco Javier and Díaz-Báñez, José},
+    year = {2010},
+    month = {01},
+    pages = {351-356},
+    title = {Characterization and Similarity in A Cappella Flamenco Cantes.}
+}
+@ARTICLE{6791736,
+    author = {E. {Gómez} and J. {Bonada}},
+    journal = {Computer Music Journal},
+    title = {Towards Computer-Assisted Flamenco Transcription: An Experimental 
+           Comparison of Automatic Transcription Algorithms as Applied to A 
+           Cappella Singing},
+    year = {2013},
+    volume = {37},
+    number = {2},
+    pages = {73-90},
+    doi = {10.1162/COMJ_a_00180}}
```

Comment:
Is there any instructions on what to cite when? Or should the user cite all the papers every time?

Reply:
Fixed. I just copied and pasted the related citations here, but now I have put the needed ones with a little comment to what they exactly refer.